packages/express-api-server/src/express-api-server.js

```diff
@@ -75,12 +77,35 @@ class ExpressApiServer extends Clonable {
     );
     this.app = expressApp.initialize();
 
-    if (port && port > 0) {
-      this.server = this.app.listen(port, () => {
-        const logger = this.container.get('logger');
-        logger.info(`${this.settings.tag} listening on port ${port}!`);
-      });
+    if (!port || port < 1) {
+      return false;
```

Comment:
if there's no valid port at this point I think is better to throw an exception to avoid creating a silent error.

Reply:
Hi @ericzon , this has been the default behaviour from the very beginning, are you sure you want to modify it?

Reply:
is it useful in any way return false at this point? the server won't be up and request won't be served. 
I usually like fail-fast approach https://www.martinfowler.com/ieeeSoftware/failFast.pdf to reduce debugging time & errors in production.

Reply:
I think the idea is to say "if (!server.start()) { ... error handling}" and if it was that way the question is ok if we want to break this by throwing an error (which is unexpected compared to before) ... but yes the better/cleaner way

Reply:
ok, let's leave it like this to respect next minor release using semver.